dll/win32/mswsock/nsplookup.c

```diff
@@ -14,8 +14,8 @@
 
 #include "mswhelper.h"
 
-#define NDEBUG
-#include <debug.h>
+#include <assert.h>
+#define ASSERT assert
```

Comment:
Not a fan of this. `msvcrt`'s assert results in annoying popups instead of just breaking into the debugger. Using it instead of `RtlAssert` may result in testbot timeouts without a backtrace, rather than an immediate debugger break that our test infrastructure knows how to handle.